docs/Protected_build_porting_guide.md

```diff
@@ -0,0 +1,363 @@
+# Protected Build Porting Guide
+
+
+## Contents
+- [Introduction](#introduction)
+	- [Prerequisites](#prerequisites)
+	- [What is Protected Build](#what-is-protected-build)
+- [Steps for porting protected build in a new chipset](#steps-for-porting-protected-build-in-a-new-chipset)
+	- [Create a new configuration for Protected Build](#create-a-new-configuration-for-protected-build)
+	- [Creating linker scripts](#creating-linker-scripts)
+	- [Userspace Initialization](#userspace-initialization)
+	- [MPU Initialization](#mpu-initialization)
+	- [Allocation of kernel and user heaps](#allocation-of-kernel-and-user-heaps)
+	- [Changes in Make files](#changes-in-make-files)
+	- [Changes in the download method](#changes-in-the-download-method)
+
+
+## Introduction
+This is a guide for porting Protected Build for a new chipset.
+
+### Prerequisites
+- TizenRT 3.0 flat build has been successfully ported and tested on the target CHIPSET board
+- Generic code relating to TizenRT protected build is already present in the codebase (Configuration variable: CONFIG_ BUILD_PROTECTED)
+- In order to bring up protected build, changes are to be made specific to the board
+- Information is available about the memory map of the board, partition information and memory organization
```

Comment:
I think we can say "Information must be gathered".
Also, please add that this information can be found in either the datasheet, configuration files, flat build ld scripts, etc.. 
Also, pls add a point mentioning that the engineer must be aware of the build and boot up process of the tizenrt flat build for the said CHIPSET.

Reply:
Done